package.json

```diff
@@ -1,12 +1,12 @@
 {
-    "name": "ts-essential-plugins",
+    "name": "typescript-essential-plugins",
```

Comment:
You changed extension id and not its name. Change displayName instead.
It'd good to change id as well, but I don't want to republish extension.